src/amplitude.js

```diff
@@ -184,9 +184,10 @@ class Amplitude {
       } else {
         this.output.connect(unit);
       }
-    } else {
-      this.output.connect(this.panner.connect(p5sound.input));
     }
+    // else {
+    //   this.output.connect(this.panner.connect(p5sound.input));
+    // }
```

Comment:
it's not clear to me why this was commented out. It looks like this is meant to be the default behavior of the `connect` method when no arguments are passed in, and I don't see that in the unit tests so it's not clear to me why this would be commented out (and better to delete it if it is meant to be commented out!)

Reply:
Yeah, I agree that it is the default behaviour. But it's just that there is no panner node for amplitude(may be removed). To remove the error, I just commented it

Do you want me to remove the lines or just post an issue?

Reply:
ah, good catch!

This sounds like a separate issue then, "Amplitude connect method with no arguments should connect to main output." 

In general, it's best not to leave commented-out code in the code base.